mathesar/api/viewsets/columns.py

```diff
@@ -22,7 +23,8 @@ class ColumnViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         table = get_table_or_404(pk=self.kwargs['table_pk'])
         sa_column_name = [column.name for column in table.sa_columns]
-        column_attnum_list = [result[0] for result in get_columns_attnum_from_names(table.oid, sa_column_name, table.schema._sa_engine)]
+        column_attnum_list = [result[0] for result in
+                              get_columns_attnum_from_names(table.oid, sa_column_name, table.schema._sa_engine)]
```

Comment:
```suggestion
        column_attnum_list = [
            result[0] for result in
            get_columns_attnum_from_names(table.oid, sa_column_name, table.schema._sa_engine)
        ]
```
This one's quite subjective, but the suggested style is more aligned with the rest of the list comprehensions in the repo.